api/v1beta1/foundationdbcluster_types.go

```diff
@@ -3016,6 +3024,11 @@ func (pClass ProcessClass) IsStateful() bool {
 	return pClass == ProcessClassStorage || pClass == ProcessClassLog || pClass == ProcessClassTransaction
 }
 
+// IsTransaction determines whether a process class is part of the transaction system.
+func (pClass ProcessClass) IsTransaction() bool {
+	return pClass == ProcessClassStateless || pClass == ProcessClassLog || pClass == ProcessClassTransaction
```

Comment:
@brownleej do you think it make sense to invert this logic to have something like: `pClass == ProcessClassStorage || pClass == ProcessClassGeneral`? Otherwise we might not replacing a Pods that services a transaction system process, like the cluster controller. Another option is to review the selection logic from FDB and add all possible roles here (except for storage).

Reply:
Yes, I think that makes sense.